module_ctx.go

```diff
@@ -737,6 +741,25 @@ func (mctx *mutatorContext) Rename(name string) {
 	mctx.rename = append(mctx.rename, rename{mctx.module.group, name})
 }
 
+// Create a new module by calling the factory method for the specified moduleType.  The module
+// will be empty, as if it were created by a blueprint file definition that contained no properties.
+// The caller should use AppendProperties
```

Comment:
This comment needs updating -- properties are passed in, not appended with AppendProperties